moveit_core/collision_detection/include/moveit/collision_detection/collision_matrix.h

```diff
@@ -94,9 +94,6 @@ class AllowedCollisionMatrix
   /** @brief Construct the structure from a message representation */
   AllowedCollisionMatrix(const moveit_msgs::AllowedCollisionMatrix& msg);
 
-  /** @brief Copy constructor */
-  AllowedCollisionMatrix(const AllowedCollisionMatrix& acm) = default;
-
```

Comment:
Aren't you a fan of explicit code? Instead of removing this explicit definition of the copy constructor, I suggest to _also_ add an explicit definition of the copy assignment operator.

Reply:
I considered that too, but to be complete we would also want to add the default move signatures as well and we rarely use the `= default` notation in this repository unless necessary.